### PR TITLE
Optional sign in with Google account

### PR DESCRIPTION
**Pull request by yivo**

### CODE REVIEW

**Comment by mod** [comment]
please rebase

File: config/initializers/omniauth.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
   provider :identity, fields: [:email], on_failed_registration: IdentitiesController.action(:new)
+
```
Comment (mod) [inline config/initializers/omniauth.rb]: I think we should activate things from application.yml configs not ENV
same for Auth0

**Review by mod — CHANGES_REQUESTED** [review]
can we add some tests ?

File: config/initializers/omniauth.rb (hunk 1 above)
Comment (yivo) [reply config/initializers/omniauth.rb]: @mod Peatio uses figaro gem for dealing with application.yml. figaro loads all values to the ENV. 

`Figaro parses a Git-ignored YAML file in your application and loads its values into ENV.`

https://github.com/laserlemon/figaro

Comment (yivo) [reply config/initializers/omniauth.rb]: @mod I can add example values to the application.yml so users will understand how to configure auth.

**Comment by yivo** [comment]
@mod 

```
Finished in 2 minutes 5.7 seconds (files took 9.07 seconds to load)
656 examples, 0 failures, 11 pending
```

**Comment by yivo** [comment]
@dkkoval Could you please take a look on it?

File: config/database.yml [hunk 2]
```diff
@@ -3,8 +3,8 @@ default: &default
   encoding: utf8
   pool: 5
   host: <%= ENV['DATABASE_HOST'] %>
-  username: <%= ENV['DATABASE_USER'] %>
-  password: <%= ENV['DATABASE_PASS'] %>
+  username: root
+  password: root
```
Comment (mod) [inline config/database.yml]: we can't take this change you need to export user / password using
export DATABASE_USER=root
export DATABASE_PASS=root

or change but not commit your database.yml

**Review by mod — CHANGES_REQUESTED** [review]
We just need to take out database.yml
Later we would need conditional auth0 or google oauth
Question @yivo can they both work at the same time ?

**Comment by yivo** [comment]
@mod Yes, they can work both as the same time. Now I will take a screenshot.

**Comment by yivo** [comment]
@mod 

screen shot 2017-11-27 at 12 36 22 pm

**Review by mod — APPROVED** [review]

**Pull request merged**